README.md

```diff
@@ -24,7 +24,7 @@ For a complete list of available packages, please see the [latest available pack
 
 ### Client: Previous Versions
 
-Last stable versions of packages that are production-ready. These libraries provide similar functionalities to the preview packages, as they allow you to use and consume existing resources and interact with them, for example: upload a storage blob. Stable library directories typically contain 'Microsoft.Azure' in their names, e.g. 'Microsoft.Azure.KeyVault'. They might not implement the [guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html) or have the same feature set as the Novemeber releases. They do however offer wider coverage of services.
+Last stable versions of packages that are production-ready. These libraries provide similar functionalities to the preview packages, as they allow you to use and consume existing resources and interact with them, for example: upload a storage blob. Stable library directories typically contain 'Microsoft.Azure' in their names, e.g. 'Microsoft.Azure.KeyVault'. They might not implement the [guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html) or have the same feature set as the November releases. They do however offer wider coverage of services.
```

Comment:
* ", since they allow you ..."
* "They do, however, offer..."

Should we also be stating "November releases"? Not only is that outdated (and will be confusing this winter), but not very helpful for customers. Maybe something along the lines of "the releases using Azure.* package names" or something along those lines.

Reply:
Yeah, that might be worth changing but I was limiting scope to fix typo 😄 
I will defer to @AlexGhiondea on whether we should make other changes or just fix typo.

Reply:
+1 to the minor tune up.     Looping in @czubair as the owner of all things documentation for additional thoughts.